backend/embedding/main.py

```diff
@@ -139,21 +146,35 @@ def main(
     filenames: list[str],
     chunk_size: int,
     chunk_overlap: int,
+    enable_partition_pdf: bool,
 ):
     exec_id = ""
     try:
         exec_id = get_exec_id()
     except Exception as e:
-        print(f"[ERROR] Failed to get exec_id: {e}")
+        logger.info(f"[ERROR] Failed to get exec_id: {e}")
         exec_id = "FAILED_TO_GET_ECS_EXEC_ID"
 
-    update_sync_status(
-        user_id,
-        bot_id,
-        "RUNNING",
-        "",
-        exec_id,
-    )
+    try:
+        update_sync_status(
+            user_id,
+            bot_id,
+            "RUNNING",
+            "",
+            exec_id,
+        )
+    except Exception as e:
+        logger.error("[ERROR] Failed to update sync status.")
+        logger.error(e)
+
+        update_sync_status(
```

Comment:
Why did you add this error handling? The situation when `update_sync_status` fails would be the dynamodb service is unavailable. In that case, we have two options:
- This task should be done with exit 1 (not continue to the following embedding process)
- Just output to the logs and continue the following processing
How do you think?

Reply:
Personally, the first option would be better to keep consistency of the status in dyanmodb (please do not confuse, this status is bot status, not the dynamodb service status) and embeddings.